packages/carequality-sdk/src/client/__tests__/carequality.test.ts

```diff
@@ -0,0 +1,179 @@
+/* eslint-disable @typescript-eslint/no-empty-function */
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+import { faker } from "@faker-js/faker";
+import { Organization } from "@medplum/fhirtypes";
+import { getEnvVar, getEnvVarOrFail } from "@metriport/shared";
+import { cloneDeep } from "lodash";
+import { APIMode, CarequalityManagementApi } from "../carequality";
+import { CarequalityManagementApiFhir } from "../carequality-fhir";
+import { getOrganizationFhirTemplate } from "./organization-template";
+
+// TODO decrease this
+// TODO decrease this
+// TODO decrease this
+// TODO decrease this
+jest.setTimeout(55_000);
+
+function getApiMode(): APIMode {
+  const apiMode = getEnvVar("API_MODE");
+  if (apiMode === "staging") return APIMode.staging;
+  if (apiMode === "dev") return APIMode.dev;
+  throw new Error(`Invalid API_MODE: ${apiMode}`);
+}
+
+function makeCarequalityManagementAPI(): CarequalityManagementApi | undefined {
+  const apiKey = getEnvVar("MANAGEMENT_API_KEY");
+  const orgCert = getEnvVar("CQ_ORG_CERTIFICATE");
+  const rsaPrivateKey = getEnvVar("CQ_ORG_PRIVATE_KEY");
+  const rsaPrivateKeyPassword = getEnvVar("CQ_ORG_PRIVATE_KEY_PASSWORD");
+  if (!apiKey || !orgCert || !rsaPrivateKey || !rsaPrivateKeyPassword) return undefined;
+  const apiMode = getApiMode();
+  return new CarequalityManagementApiFhir({
+    apiKey,
+    apiMode,
+    orgCert,
+    rsaPrivateKey,
+    rsaPrivateKeyPassword,
+  });
+}
+
+type OrgWithId = Organization & Required<Pick<Organization, "id">>;
+
+// TODO Need to setup CICD to support running these automatically
```

Comment:
why would we run these tests in production? Wouldn't we just run them in CI?

Reply:
We run E2E test against `staging` and `production` as part of the CICD build